src/pyOpenMS/tests/unittests/test_core_structures.py

```
@@ -0,0 +1,768 @@
+    f.setQuality(2.0)
+    f.setWidth(4.0)
+    assert f.getCharge() == 1
+    assert f.getQuality() == 2.0
+    assert f.getWidth() == 4.0
+
+    f.insert(0, pyopenms.Peak2D(), 1)
+    f.insert(1, pyopenms.BaseFeature())
+    f.insert(2, pyopenms.ConsensusFeature())
+
+    f.computeConsensus()
+    f.computeDechargeConsensus
```

Ruff (0.14.10)

Found useless expression. Either assign it to a variable or remove it.

(B018)